oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -54,6 +55,8 @@ pub struct RuntimeConfiguration {
     pub grpc_config: GrpcConfiguration,
     /// Application configuration.
     pub app_config: ApplicationConfiguration,
+    /// Start-of-day configuration to feed to the running Application.
```

Comment:
Maybe mention here too that if one is not specified, a default one will be sent anyways?

Which makes me think, maybe it will more more explicit to remove the `Option` here and require the creator of this struct to provide a `ConfigMap`, which may at that point be the default one? Rather than having to document what happens in the `None` case here and throughout the internals of the Runtime.

Fine for me either way.